jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/logging/PlainConsoleLogger.java

```diff
@@ -49,7 +49,9 @@ public void log(Level logLevel, String message) {
     }
     Consumer<String> messageConsumer = messageConsumers.get(logLevel);
 
-    singleThreadedExecutor.execute(() -> messageConsumer.accept(message));
+    // remove the color from the message
+    final String plainMessage = message.replaceAll("\u001B\\[[0-9;]{1,5}m", "");
```

Comment:
I think this is too lenient and actually more confusing. For example, `[0-9;]{1,5}` accepts a string like `;;8;3;`. I thin there can only be one `;`.

Reply:
Oh, actually, it turns out [there can be multiple semi-colons](https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_(Select_Graphic_Rendition)_parameters).

> Several attributes can be set in the same sequence, separated by semicolons.

Therefore, `\u001B\\[[0-9;]*m` is more accurate. However, I am fine limiting the scope by finding up to 5 characters.